sql/parser/builtins.go

```diff
@@ -331,12 +334,23 @@ var builtins = map[string][]builtin{
 			types:      typeList{},
 			returnType: DummyFloat,
 			impure:     true,
-			fn: func(args DTuple) (Datum, error) {
+			fn: func(_ EvalContext, args DTuple) (Datum, error) {
 				return DFloat(rand.Float64()), nil
 			},
 		},
 	},
 
+	"unique_id": {
```

Comment:
Yeah, we should avoid "uuid" if it's not one of the standard forms of UUID. "K-sorted" is too obscure (I've never heard the term before, and googling it isn't much help).